packages/webpack-stats-differ/src/getFriendlyAssetName.ts

```diff
@@ -44,9 +44,10 @@ const hashRegex = /_([a-z0-9]{20})/;
 export const removeHashFromName = (name: string): string => {
```

Comment:
What am I missing that prevents this method from being merely:

```typescript
export const removeHashFromName = (name: string): string => {
 return name.replace(/([_.][a-z0-9]{20})(?:\b)/, "");
};
```

This regex and replace will remove the 20 chars hash and the `"."` or `"_"` separator before it from the name.

Reply:
Then we don't need to import identity or upath, and thus need to check if we can remove them from the list of deps in the package.json